solr/solrj/src/java/org/apache/solr/common/util/StrUtils.java

```diff
@@ -363,4 +366,16 @@ public static String formatString(String pattern, Object... args) {
   public static boolean isNullOrEmpty(String string) {
     return string == null || string.isEmpty();
   }
+
+  public static String stringFromReader(Reader inReader) throws IOException {
+    try (Reader reader = new BufferedReader(inReader)) {
+      char[] arr = new char[8 * 1024];
+      StringBuilder buffer = new StringBuilder();
```

Comment:
triple buffering?!

Reply:
instead I recommend inReader.transferTo(StringWriter)

Reply:
thanks looks a lot nicer - fixed in b8e3fd33d327d0fd0c57ea226a4ea90764c3298f